ee-9/feature-pack/pom.xml

```diff
@@ -756,6 +756,10 @@
                     <groupId>org.wildfly.wildfly-http-client</groupId>
                     <artifactId>wildfly-http-transaction-client</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>${ee.maven.groupId}</groupId>
+                    <artifactId>wildfly-transactions-jakarta</artifactId>
+                </exclusion>
```

Comment:
@yersan  I pushed the update but I still get the same issue after making that change.

Reply:
I missed two of your comments, I am retesting after addressing those.

Reply:
@yersan But I still get the error. Do you still have your branch where you managed to get it working, if so I can clone that and use it to compare with my updates after addressing your comments?

Reply:
@mmusgrov The build succeeds if you place the missing dependencies before `wildfly-iiop-openjdk`, which indeed is wrong, we should use `wildfly-iiop-openjdk-jarkarta`.

Having said that, there is still additional work to do:

- Use `wildfly-iiop-openjdk-jarkarta` instead of `wildfly-iiop-openjdk`
- Exclude whatever is not needed from the `wildfly-ee-feature-pack-common`, e.g `narayana-jts-integration`,`narayana-jts-idlj` and `wildfly-iiop-openjdk`. I've noticed they are still being transformed, but they shouldn't.

Let me know if you want me to take a closer look at all of this.

Reply:
@yersan 
Which missing dependencies?
Which dependencies depend on the order of declaration?

And the build still fails with the same error `WFLYCTL0153: No META-INF/services/org.jboss.as.controller.Extension found for org.jboss.as.transactions` after addressing your first two bullets. These extra requirements are not mentioned in Brian's document so I am just guessing at these extra items you say I need.

I would take you up on your offer of "taking a closer look at all of this" but, since I am just guessing at these extra items I will have the same problems with the other systems I have pending updates for, namely `wildfly-rts-jakarta` and `wildfly-xts-jakarta`, and I suspect your good will won't extend that far.